app/views/profiles/edit.html.erb

```diff
@@ -1,7 +1,13 @@
-<%= render 'shared/error_messages', object: @profile %>
+<%= render 'shared/error_messages', object: @profile_form %>
 
 <h1><%= t('edit_profile') %></h1>
 
-<%= render 'form' %>
+<%= form_for(@profile) do |f| %>
+  <%= render 'profiles/form_partials/profile_details', f: f %>
+
+  <div class="actions">
+    <%= f.submit %>
+  </div>
+<% end %>
```

Comment:
It would be better if we used ProfileForm for both new and edit actions to be consistent.
This should also dry up the new and edit view templates as they should be the same (ie fields available when creating a profile are available when editing a profile